### PR TITLE
BOM-1103

### PR DESCRIPTION
**Pull request by awais786**
Adding env vars for django2.2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,22 @@
 language: python
 python:
 - 2.7
-- 3.6
+- 3.5
 env:
 - TOXENV=django111
 - TOXENV=django20
+- TOXENV=django21
```
Comment (Ayub-Khan) [inline .travis.yml]: where are TOXENV defined?

Comment (awais786) [reply .travis.yml]: I have added this question on other ticket also.

Comment (jmbowman) [reply .travis.yml]: The Travis configuration here doesn't really do anything except deploy to PyPI on tags (the test script is `echo "Success"`).  There's no tox configuration, and not even any unit tests.  So there isn't much point in testing multiple Python or Django versions here, there just aren't any test cases for them to make a difference on.

**Comment by awais786** [comment]
@jmbowman Please check. Does it make sense to release a new version with these changes ?

**Pull request merged**